source/assets/css/_imports/des-guide.less

```diff
@@ -0,0 +1,346 @@
+    }
+
+    h2 {
+        font-size: 22px;
+        margin: 2px 0;
+    }
+
+    h3 {
+        font-size: 33px;
+        font-weight: 600;
+    }
+
+    .styletile-headline {
+        padding: 2% 1% 0 1%;
+    }
+
+    .udemy-teaser--screenshot {
+        background: url("../../img/screen-des-udemy.jpg") top center no-repeat;
+        background-size: contain;
+        height: 170px;
+        margin: 0 auto;
+        width: 278px;
+    }
+
+    .styletile--screenshot {
+        background: url("../../img/screen-des-styletile.svg") top center no-repeat;
+        background-size: contain;
+        margin: 0 auto;
+        height: 170px;
+        width: auto;
+    }
+
+    .styletile--dialog {
+        text-align: left;
+        padding: 0 0 20px 20px;
+        clear: both;
+
+        .avatar-one {
+            background: url("../../img/styletile-preview.svg") center center no-repeat;
+            background-size: cover;
+            float: left;
+            height: 105px;
+            width: 80px;
+        }
+
+        .avatar-two {
+            background: url("../../img/styletile-cheatsheet.svg") center center no-repeat;
+            background-size: cover;
+            margin-top: 10px;
+            float: left;
+            height: 60px;
+            width: 80px;
+        }
+
+        h4 {
+            color: darken(@text-color, 20%);
+            font-size: 24px;
+        }
+
+        .chat {
+            color: @text-color;
+            display: block;
+            font-weight: 500;
+            padding-left: 120px;
+
+        }
+    }
+}
+
+/*------- BUTTON STYLES -------*/
+.start-guide {
+    .button {
+        clear: both;
+        margin: 0 auto;
+        text-align: center;
+        padding-top: 10px;
+
+        a {
+            background: @gray05;
+            border-radius: 30px;
+            box-shadow: 0 0.5rem 1rem -0.25rem rgba(0, 0, 0, 0.12);
+            color: #FFFFFF;
+            display: inline-block;
+            font-weight: 600;
+            font-size: 16px;
+            padding: 13px 25px;
+            transition: all .3s cubic-bezier(0.25, 0.2, 0.45, 1);
+
+            &:hover {
+                background: darken(@gray05, 9%);
+                box-shadow: 0 0.5rem 1rem -0.25rem rgba(0, 0, 0, 0.33);
+                transform: translateY(-3px);
+                -webkit-transform: translateY(-3px);
+                -moz-transform: translateY(-3px);
+                -ms-transform: translateY(-3px);
+                -o-transform: translateY(-3px);
+            }
+        }
+    }
+
+    .udemy-teaser .button a {
+        background: #FA6179;
+        box-shadow: 0 0.5rem 1rem -0.3rem rgba(250, 97, 121, 0.22);
+        color: #FFF;
+
+        &:hover {
+            background: darken(#FA6179, 5%);
+            box-shadow: 0 0.5rem 1rem -0.3rem rgba(250, 97, 121, 0.44);
+        }
+    }
+}
+
+/*------- MENUE BOXES -------*/
+.des-guide--menu {
+    display: inline-block;
+    width: 100%;
+
+    .row {
```

Comment:
You're defining `.row` multiple times and it has the exact same CSS properties assigned. Please refactor it.

Reply:
check.